inngest/src/main/kotlin/com/inngest/Inngest.kt

```diff
@@ -21,37 +19,28 @@ class Inngest
 
         internal val httpClient = HttpClient(RequestConfig(headers))
 
-        inline fun <reified T> send(payload: Any): T? =
-            sendEvent<T>(payload) lambda@{ response ->
+        /**
+         * Send a single event to Inngest,
+         *
+         * @param event The event to send.
+         *
+         */
+        fun send(event: InngestEvent): SendEventsResponse? = send(arrayOf(event))
```

Comment:
do other SDKs use the plural response type even if a singular event is sent?

Reply:
both do yeah.